src/layout/DropZone.tsx

```diff
@@ -0,0 +1,144 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { MdCloudUpload } from 'react-icons/md';
+
+export default function DropZone(props?: { color?: string; width?: string }) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [alert, setAlert] = useState('');
+  const [active, setActive] = useState(false);
+  const onDrop = useCallback((acceptedFiles) => {
+    // Do something with the files
+    acceptedFiles.forEach((file: Blob) => {
+      const reader = new FileReader();
+
+      reader.onabort = () => setAlert('file reading was aborted');
+      reader.onerror = () => setAlert('file reading has failed');
+      reader.onload = () => {
+        setActive(true);
+        // Do whatever you want with the file contents
+        //const binaryStr = reader.result;
+        setAlert('file uploaded');
+        //console.log(binaryStr);
+      };
+      reader.readAsArrayBuffer(file);
+    });
+  }, []);
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+
+  return (
+    <>
+      <div
+        {...getRootProps()}
+        css={css`
+          overflow: hidden;
+          position: relative;
+          width: ${props ? props.width : '100%'};
+          min-width: 400px;
+          min-height: 300px;
```

Comment:
Avoid adding too many default styles to the component. It's better to let the user of the component add their own styles.

Reply:
deleted